src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/CreateSomaticPanelOfNormals.java

```diff
@@ -181,14 +183,22 @@ private static final int altCount(final Genotype g) {
     }
 
     private static final double germlineProbability(final double alleleFrequency, final int altCount, final int totalCount) {
+        if (alleleFrequency < NEGLIGIBLE_ALLELE_FREQUENCY) {
+            return 0;
+        }
+
         final double hetPrior = alleleFrequency * (1 - alleleFrequency) * 2;
         final double homPrior = MathUtils.square(alleleFrequency);
         final double hetLikelihood = MathUtils.binomialProbability(totalCount, altCount, 0.5);
         final double homLikelihood = MathUtils.binomialProbability(totalCount, altCount, 0.98);
 
         final double artifactLikelihood = new BetaBinomialDistribution(null, ARTIFACT_ALPHA, ARTIFACT_BETA, totalCount).probability(altCount);
 
-        return MathUtils.normalizeFromRealSpace(new double[] {hetPrior * hetLikelihood + homPrior * homLikelihood, ARTIFACT_PRIOR * artifactLikelihood})[0];
+        final double[] relativeProbsOfHetHomArtifact = {hetPrior * hetLikelihood + homPrior * homLikelihood, ARTIFACT_PRIOR * artifactLikelihood};
+        if (MathUtils.sum(relativeProbsOfHetHomArtifact) < 0) {
+            return 0;
+        }
+        return MathUtils.normalizeFromRealSpace(relativeProbsOfHetHomArtifact)[0];
```

Comment:
The indices look like they have special meaning.  Can you describe, in a comment, the meaning of the values in [0] and [1]?

Reply:
done